pkg/skr/skr.go

```diff
@@ -117,7 +116,7 @@ func SecureKeyRelease(identity common.Identity, certState attest.CertState, skrK
 		return nil, errors.Wrapf(err, "releasing the key %s failed", skrKeyBlob.KID)
 	}
 
-	logrus.Debugf("Key Type: %s Key %v", kty, keyBytes)
+	logrus.Debugf("Key Type: %s", kty)
```

Comment:
Can we add something to say that the key was successfully released? I'm pretty sure the key type is logged in several places before this so this log does nothing to tell us where in the code we are if we need to debug something.